tools/cockpit.spec

```diff
@@ -347,8 +347,8 @@ Cockpit support for reading PCP metrics and loading PCP archives.
 %package dashboard
 Summary: Cockpit SSH remoting and dashboard
 Requires: libssh >= %{libssh_version}
-Requires: cockpit-ws = %{version}-%{release}
-Provides: cockpit-ssh = %{version}-%{release}
+Requires: cockpit-bridge > 134
```

Comment:
Oh duh. This was about the ```cockpit-dashboard``` not about ```cockpit-system``` ... and yes, the ```manifest.json``` declares the ```"bridges"``` which in turn require that dependency. So updating the ```manifest.json``` makes sense here.